qemu/tests/win_sigverif.py

```diff
@@ -22,6 +23,8 @@ def run(test, params, env):
 
     vm = env.get_vm(params["main_vm"])
     session = vm.wait_for_login()
+    vm.send_key('meta_l-d')
+    time.sleep(60)
```

Comment:
Shouldn't we wait 60s first, then send key?

Reply:
Actually,the order doesn't matter.